nodejs/week2/mealSharing/src/bakckend/routes/meals.js

```diff
@@ -0,0 +1,83 @@
+const express = require("express");
+const router = express.Router();
+const meals = require("../data/reviewed-meals");
+
+router.get("/", (req, res) => {
+	const maxPrice = req.query.maxPrice;
+	const title = req.query.title;
+	const createdAfter = req.query.createdAfter;
+	const limit = req.query.limit;
+
+	if (maxPrice) {
+		const maxPriceValue = parseInt(maxPrice);
```

Comment:
you parsed the price value to a number. Nice! But you are still not checking whether it is a number or not. 

What happens if you try this? parseInt('hello world')

Reply:
or if a user sends a request to `/meals?maxPrice=hello`